src/noun.substance

```diff
@@ -505,6 +526,10 @@ w: californium
 d: a radioactive transuranic element; discovered by bombarding curium with alpha particles
 hypo: metal
 
+w: Chalk
+d: [lithology] A light-coloured (white-gray) marine limestone composed almost entirely of fine crystalline calcite. These porous limestones consist of foraminifera and calcareous algae, and usually contain chert nodules (Neuendorf et al., 2005).
```

Comment:
note que temos o synset http://wn.mybluemix.net/synset?id=06121375-n, então podemos ligar estes novos synsets à este via alguma relação... talvez https://wordnet.princeton.edu/documentation/wninput5wn (;c    Domain of synset - TOPIC)? Veja arquivo no own-en/conf/pointer-mnemonics.txt

Reply:
Onde está esse arquivo pointer-mnemonics.txt?

Reply:
Ok, will add ;c to geology and lithology and -c for the members of such domains.

Reply:
vc não vai usar o `;c` mas sim `dt:` idem para `-c` que deve ser `mt`.

Reply:
no repositório own-en diretório `conf`

Reply:
Nesse onde estamos? Olhei em todos os branches e pastas, não há past conf (apenas config) e não há arquivo txt. Procurei no own-pt com a busca e também não encontrei. Alguma ideia de onde possa estar?

Reply:
https://github.com/own-pt/own-en/blob/master/config/relations.tsv

Reply:
Obrigado!